kbfsgit/runner.go

```diff
@@ -311,6 +311,41 @@ func (r *runner) initRepoIfNeeded(ctx context.Context, forCmd string) (
 	return repo, fs, nil
 }
 
+func percent(n int64, d int64) float64 {
+	return float64(100) * (float64(n) / float64(d))
+}
+
+func humanizeBytes(n int64, d int64) string {
+	const kb = 1024
+	const kbf = float64(kb)
+	const mb = kb * 1024
+	const mbf = float64(mb)
+	const gb = mb * 1024
+	const gbf = float64(gb)
+	// Special case the counting of bytes, when there's no denominator.
+	if d == 1 {
+		if n < kb {
+			return fmt.Sprintf("%d bytes", n)
+		} else if n < mb {
+			return fmt.Sprintf("%.2f KB", float64(n)/kbf)
```

Comment:
Since we are using K/M for object counting below to represent 1000 and 10^6, do you think we should use `KiB` (and below in other places) here to make it clear it's 1024 rather than 1000 times or previous level?

Reply:
I really dislike MiB even though it's more correct.  For me, MB = 1024*1024, no matter what anyone says.